VCFv4.3.tex

```diff
@@ -503,7 +505,15 @@ \subsubsection{Genotype fields}
   All phased genotypes that do not contain a PS subfield are assumed to belong to the same phased set.
   If the genotype in the GT field is unphased, the corresponding PS field is ignored.
   The recommended convention is to use the position of the first variant in the set as the PS identifier (although this is not required).
-\end{itemize}
+  \item LAA (and LAD and LPL (*):
```

Comment:
I would follow the existing of one item per field, otherwise it may be confusing to readers searching for LAD and LPL.

Reply:
should I leave the paragraph as is, and add entries for LAD and LPL pointing to LAA?

Reply:
The AD, ADF, ADR paragraph also groups related items into a single paragraph. Fortuitously in both cases the three entries are adjacent alphabetically.

(Though the whole LAA/LAD/LPL paragraph wants to move up to between HQ and MQ, and this label (`LAA (and LAD and LPL (*)`) wants less ‘and’ and better balanced parentheses!)